.flake8

```diff
@@ -11,9 +11,14 @@ ignore = D205,D212,D415,E203,W503
 # https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length
 max-line-length = 88
 
+# This is for pydocstyle - to allow docstrings in __init__ functions
+# This rule was used for easier migration from darglint
+# TODO: remove this rule
+allow-init-docstring=True
```

Comment:
Is it possible to remove this rule within the scope of this PR ?

Reply:
It would make the diff gigantic and much harder to review.

Reply:
hmmm